templates/shared/search/code/results.tmpl

```diff
@@ -0,0 +1,44 @@
+<div class="flex-text-block gt-fw">
+	{{$pageURL := print $.SourcePath "/search"}}
+	{{if and $.PageIsExploreCode $.ContextUser}}
+		{{$pageURL = print AppSubUrl "/" $.ContextUser.Name "/-/code"}}
+	{{else if $.PageIsExploreCode}}
+		{{$pageURL = print AppSubUrl "/explore/code"}}
+	{{end}}
+	{{range $term := .SearchResultLanguages}}
+	<a class="ui {{if eq $.Language $term.Language}}primary{{end}} basic label gt-m-0"
+		href="{{$pageURL}}?q={{$.Keyword}}{{if ne $.Language $term.Language}}&l={{$term.Language}}{{end}}&fuzzy={{$.IsFuzzy}}">
+		<i class="color-icon gt-mr-3" style="background-color: {{$term.Color}}"></i>
+		{{$term.Language}}
+		<div class="detail">{{$term.Count}}</div>
+	</a>
+	{{end}}
+</div>
+<div class="repository search">
+	{{range $result := .SearchResults}}
+		{{$repoLink := $.SourcePath}}
+		{{if $.PageIsExploreCode}}{{$repoLink = (index $.RepoMaps .RepoID).Link}}{{end}}
```

Comment:
It also looks fragile here and below (by using `if`), that's what I meant "over-engineering" .....

Reply:
If there are so many different details, I think it's better to use 2 separate template files, instead of mixing the "if" logics into one template file.

Reply:
SImplified just to decide between .Repo or $.RepoID in the beginning